public_html/common/scss/common.scss

```diff
@@ -0,0 +1,67 @@
+body {
+
+  position: relative;
+  min-height: 100vh;
+
+  // Scoll bar
+  scrollbar-width: thin;
+  scrollbar-color: #9b9b9b #f1f1f1;
+
+  &::-webkit-scrollbar {
+    width: 12px;
+  }
+
+  &::-webkit-scrollbar-track {
+    background: #f1f1f1; 
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background-color: #9b9b9b;
+    border-radius: 20px;
+    border: 2.6px solid #f1f1f1;
+
+    &:hover {
+      background-color: #868686;
+    }
+    &:active {
+      background-color: #696969;
+    }
+  }
```

Comment:
Could you provide screenshots to illustrate the before-and-after looks of the scrollbar UI? I notice that the current scrollbar won't auto-hide when inactive, which is the default behavior. But I didn't notice other changes.

Reply:
## Old scroll bar
![old scroll bar](https://user-images.githubusercontent.com/44086040/84193298-1256b780-aa50-11ea-982e-6189bbdcd3ca.PNG)

## New scroll bar
![new scroll bar](https://user-images.githubusercontent.com/44086040/84193314-1a165c00-aa50-11ea-86fe-b92cd584c137.PNG)

Could you elaborate on the auto-hide feature? I was never aware of that feature in the first place. Perhaps I could add it on with a few extra lines.

Reply:
That's interesting. 

The following 3 screenshots are Chrome, Safari, and FireFox on macOS:

<img width="525" alt="Screen Shot 2020-06-09 at 2 56 15 PM" src="https://user-images.githubusercontent.com/19198708/84194606-e133b280-aa62-11ea-9c2b-65eee9e4ed42.png">
<img width="525" alt="Screen Shot 2020-06-09 at 2 56 59 PM" src="https://user-images.githubusercontent.com/19198708/84194612-e3960c80-aa62-11ea-8cdf-97f68a474948.png">
<img width="525" alt="Screen Shot 2020-06-09 at 2 58 39 PM" src="https://user-images.githubusercontent.com/19198708/84194616-e55fd000-aa62-11ea-8a3e-71a5c84d973a.png">

Reply:
Screenshots of Chrome (left) and FireFox (right) on Ubuntu:

![image](https://user-images.githubusercontent.com/19198708/84195292-ed6c3f80-aa63-11ea-8e8f-74498ee27323.png)